### ISSUE TITLE
Hi there

### ISSUE THREAD

**Issue by Athanasius** [talkers/summink#1]
I was today pondering slinging the actual current (like it's been worked on any time in the last ... two decades) Surfers source on GH, but thought to check if anyone had already put Summink, or a derivative, here.

Absolutely no objections, I just found it amusing, *and* you left Issues active !

**Comment by robn** [comment]
Yes I definitely deliberately left issues open in case you came calling!

Posting the Surfers code would be vaguely interesting for at least three people, I'm sure :)

Anyway, nice to meet you o/
